product_docs/docs/pgd/4/harp/index.mdx

```diff
@@ -242,3 +242,53 @@ location: dcb
 This applies to any HARP Proxy nodes that are designated in those respective
 data centers as well.
 
+### Health check
+
+HARP provides the following liveness and readiness health check APIs. These are HTTP(s) APIs and if enabled HARP will start a HTTP server within the respective service. Both the API  endpoints are HTTP(s) `GET` requests and do not require a request body.
+```
+GET /health/is-live
+GET /health/is-ready
+```
+
+#### Liveness
+
+Liveness endpoint returns `true` with HTTP status code `200 (OK)` if the HARP service is up and a connection refused error if the service is down but will never return `false`. This is due to the fact that the HTTP server listening for request is stopped if the corresponding HARP service is down. This API endpoint is available for `harp-manager` and `harp-proxy` services.
+
+#### Readiness
+
+Readiness endpoint is exposed for `harp-proxy` only. On receiving this request, proxy executes server check query through its own listening address and port (default 6432), like any client application. The query is executed on the current write leader if it is available. If the query returns successfully then API returns `true` with HTTP status code `200 (OK)` to HTTP client  else `false` with HTTP status code `500 (Internal Server Error)`. It would return `false` in the following scenarios.
+* The HARP Proxy cannot establish a TCP connection to the DCS
+* The DCS reports that there is no write leader available
+* The DCS reports that it is unhealthy like no Raft leader
+* The HARP Proxy cannot establish a TCP connection to the current write leader
+* The TLS certificate on the current write leader is expired
+
+It is important to know that once started proxy always remains in available state even if this endpoint return `false` because the proxy can recover itself automatically (self heal) once the BDR or DCS side issues are resolved. If `harp-proxy` service is down then, this too would return connection refused error.
+
+!!! Note
```

Comment:
no need to add this...I think we should be abstract about the implementation here because it may change. 

just add API endpoints, their responses. and some details about configuration.